data/net/minecraft/data/worldgen/placement/PlacementUtils.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/data/worldgen/placement/PlacementUtils
+	METHOD filtered (Lnet/minecraft/world/level/levelgen/feature/Feature;Lnet/minecraft/world/level/levelgen/feature/configurations/FeatureConfiguration;Lnet/minecraft/world/level/levelgen/blockpredicates/BlockPredicate;)Lnet/minecraft/core/Holder;
+		ARG 0 feature
+		ARG 1 featureConfiguration
+	METHOD filteredByBlockSurvival (Lnet/minecraft/world/level/block/Block;)Lnet/minecraft/world/level/levelgen/placement/BlockPredicateFilter;
+		ARG 0 block
+	METHOD inlinePlaced (Lnet/minecraft/core/Holder;[Lnet/minecraft/world/level/levelgen/placement/PlacementModifier;)Lnet/minecraft/core/Holder;
+		ARG 0 feature
+		ARG 1 modifiers
+	METHOD inlinePlaced (Lnet/minecraft/world/level/levelgen/feature/Feature;Lnet/minecraft/world/level/levelgen/feature/configurations/FeatureConfiguration;[Lnet/minecraft/world/level/levelgen/placement/PlacementModifier;)Lnet/minecraft/core/Holder;
+		ARG 0 feature
+		ARG 1 featureConfiguration
+		ARG 2 modifiers
+	METHOD onlyWhenEmpty (Lnet/minecraft/world/level/levelgen/feature/Feature;Lnet/minecraft/world/level/levelgen/feature/configurations/FeatureConfiguration;)Lnet/minecraft/core/Holder;
+		ARG 0 feature
+		ARG 1 featureConfiguration
+	METHOD register (Ljava/lang/String;Lnet/minecraft/core/Holder;Ljava/util/List;)Lnet/minecraft/core/Holder;
+		ARG 0 name
+		ARG 1 feature
+		ARG 2 modifiers
+	METHOD register (Ljava/lang/String;Lnet/minecraft/core/Holder;[Lnet/minecraft/world/level/levelgen/placement/PlacementModifier;)Lnet/minecraft/core/Holder;
+		ARG 0 name
+		ARG 1 feature
+		ARG 2 modifiers
```

Comment:
```suggestion
		ARG 2 placement
```
Not plural since this takes a single, non-varadic argument of the already built placement list.